docs/reference/packetbeat/setup-repositories.md

```diff
@@ -17,10 +17,112 @@ to sign all our packages. It is available from [https://pgp.mit.edu](https://pgp
 
 ## APT [_apt]
 
-Version 9.0.0-beta1 of Beats has not yet been released.
+To add the Beats repository for APT:
+
+1. Download and install the Public Signing Key:
+
+    ```shell
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+    ```
+
+2. You may need to install the `apt-transport-https` package on Debian before proceeding:
+
+    ```shell
+    sudo apt-get install apt-transport-https
+    ```
+
+3. Save the repository definition to  +/etc/apt/sources.list.d/elastic-{{major-version}}.list+:
```

Comment:
```suggestion
3. Save the repository definition to +/etc/apt/sources.list.d/elastic-{{major-version}}.list+:
```

Reply:
Thanks! I also changed those strange `+` signs to backticks.